docs/database/migration-guide.md

```diff
@@ -0,0 +1,79 @@
+to
+[v1.TODO (stable)](https://github.com/open-telemetry/semantic-conventions/blob/v1.TODO/docs/database/README.md).
+
+### Database client span attributes
+
+<!-- prettier-ignore-start -->
+| Change                                          | Comments                                                                                                    |
+|-------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| `db.connection_string`                          | Removed                                                                                                     |
+| `db.user`                                       | Removed                                                                                                     |
+| `network.transport`                             | Removed                                                                                                     |
+| `network.type`                                  | Removed                                                                                                     |
+| `db.name` &rarr; `db.namespace`                 |                                                                                                             |
+| `db.redis.database_index` &rarr; `db.namespace` |                                                                                                             |
+| `db.mssql.instance_name`                        | Incorporated into the new `db.namespace`                                                                    |
+| `db.instance.id`                                | Removed, replaced by `server.address` or `db.namespace` as appropriate                                      |
+| `db.statement` &rarr; `db.query.text`           | Clarified, SHOULD be collected by default only if there is sanitization that excludes sensitive information |
+| `db.operation` &rarr; `db.operation.name`       |                                                                                                             |
+| `db.sql.table` -> `db.collection.name`          |                                                                                                             |
+| `db.cassandra.table` -> `db.collection.name`    |                                                                                                             |
+| `db.mongodb.collection` -> `db.collection.name` |                                                                                                             |
+| `db.cosmosdb.container` -> `db.collection.name` |                                                                                                             |
+| New: `db.query.parameter.<key>`                 | Opt-In                                                                                                      |
```

Comment:
nit: 

maybe 
```suggestion
| `db.query.parameter.<key>`                      | New, Opt-In                                                                                                      |
```

to be consistent with 

| `db.connection_string` | Removed

?

Reply:
the logic(?) behind the current format is that the column normally starts with the old name, but in this case there's no old name, so I put "New: " in front of it